client/src/components/Companies/Scores/Scores.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import "../../../styles/Companies.css";
+import DiversityImg from "../../../assets/diversity.png";
+import WorkerExploitImg from "../../../assets/workerexploit.png";
+import WasteImg from "../../../assets/wastepollution.png";
+import SustainableImg from "../../../assets/sustainable.png";
+import axios from "axios";
+import BrandPerformance from "./BrandPerformance";
+
+
+const companyTotalScores = [
+  {
+    A: 0,
+    B: 0,
+    C: 0,
+    D: 0,
+  }
+]
+
+
+export default function Scores(props) {
+  const [TotalScore, setTotalScore] = React.useState(companyTotalScores);
+
+  useEffect(()=>{
+    axios.get('/companyTotalScores',{params: props.company}).
+    then((resp) => {
+      let data = TotalScore;
+      data[0]['A'] = resp.data[0]["Ascore"];
```

Comment:
why continue to use 2-D array here? Don't just copy the old code.   One dimensional array is enough,  don't add unnecessary complexity.